### DISCUSSION TITLE
Accessing STDERR sender internal?

### DISCUSSION THREAD

**Discussion by stevepentland** [klogging/klogging#139] [Q&A] [answered]
Hi, I am trying to use the DSL configuration style when setting up logging. However when I try to do the following (from the docs at https://klogging.io/docs/configuration/dsl#a-more-complex-example) I am unable to access STDERR
```
sink("stderr", RENDER_SIMPLE, STDERR)
```
and it appears it is internal in `STDERR.kt`. What am I doing wrong?

**Comment by mjstrasser** [comment, marked as answer]
You are doing nothing wrong.

See #140 for fix.
